math-scala/src/main/scala/org/apache/mahout/math/drm/package.scala

```diff
@@ -80,9 +80,6 @@ package object drm {
   /** Just throw all engine operations into context as well. */
   implicit def ctx2engine(ctx: DistributedContext): DistributedEngine = ctx.engine
 
-  implicit def drm2drmCpOps[K: ClassTag](drm: CheckpointedDrm[K]): CheckpointedOps[K] =
```

Comment:
Hm. indeed. ok.


Normally we import both and i'd say drm._ is the place for it. RLikeDrmOps
deals with R dialect specifically so i'd say if i had to remove it, i'd
remove it from RLikeDrmOps (assuming all tests pass).


On Fri, Feb 13, 2015 at 2:05 PM, Erlend Hamnaberg <notifications@github.com>
wrote:

> In math-scala/src/main/scala/org/apache/mahout/math/drm/package.scala
> <https://github.com/apache/mahout/pull/74#discussion_r24700771>:
>
> > @@ -80,9 +80,6 @@ package object drm {
> >    /** Just throw all engine operations into context as well. */
> >    implicit def ctx2engine(ctx: DistributedContext): DistributedEngine = ctx.engine
> >
> > -  implicit def drm2drmCpOps[K: ClassTag](drm: CheckpointedDrm[K]): CheckpointedOps[K] =
>
> it already exits here:
> https://github.com/apache/mahout/blob/master/math-scala/src/main/scala/org/apache/mahout/math/drm/RLikeDrmOps.scala#L137.
> if you upgrade to 2.11 this would crash. and both are imported into scope.
> Why this does not crash on 2.10 i have no idea.
> Not sure which one that should be removed, but I chose this one.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/mahout/pull/74/files#r24700771>.
>